test/state/player_state_test.cpp

```diff
@@ -161,3 +161,102 @@ TEST_F(PlayerStateTest, GetOffsetFromPosition) {
     EXPECT_EQ(offset1, Vec2D(2, 2));
     EXPECT_EQ(offset2, Vec2D(2, 2));
 }
+
+TEST_F(PlayerStateTest, TowerTest) {
+    player_state::Tower tower1(10);
+    player_state::Tower tower2(19);
+    player_state::Tower tower3(tower1);
+    player_state::Tower tower4;
+    tower4.id = 19;
```

Comment:
Not this. Use bots and towers declared in this GTest constructor. See how the previous test were written